### DISCUSSION TITLE
Full Vault Sync not working for me

### DISCUSSION THREAD

**Discussion by pplebel** [thesamim/TickTickSync#149] [Q&A]
Hi everyone,

I am sure I am missing something, but when I enable Full Vault Sync, and I add a task in a file, nothing happen. Even when I create a task with the `#ticktick` nothing happens. What am I missing?

**Comment by thesamim** [comment]
@pplebel : When full vault sync is enabled, the next time a Sync happens it will scan all your files for tasks not already added to TickTick and add them to TickTick, and tasks updated in TickTick will be downloaded to Obsidian.

Sync happens every five minutes by default. You can change that timing in settings.
Or you can sync manually from settings
Or you can set "Debug Mode" on and have a Sync Icon in your left ribbon and sync manually whenever you want.

If a Sync happened, and you're still not seeing your tasks added to Ticktick (and generally syncing happening), we'll need to take a look at your settings to see what might be missing.

**Comment by thesamim** [comment]
Assuming this is sorted. Please open a new issue if it's not.
